doc/classes/ResourceImporterTexture.xml

```diff
@@ -65,6 +65,62 @@
 		<member name="mipmaps/limit" type="int" setter="" getter="" default="-1">
 			Unimplemented. This currently has no effect when changed.
 		</member>
+		<member name="process/channel_remap/alpha" type="int" setter="" getter="" default="3">
+			Specifies the data source of the image's alpha channel.
+			[b]Red:[/b] Use the values from the red channel.
+			[b]Green:[/b] Use the values from the green channel.
+			[b]Blue:[/b] Use the values from the blue channel.
+			[b]Alpha:[/b] Use the values from the alpha channel.
+			[b]Red Inverted:[/b] Use inverted values from the red channel ([code]1.0 - R[/code]).
+			[b]Green Inverted:[/b] Use inverted values from the green channel ([code]1.0 - G[/code]).
+			[b]Blue Inverted:[/b] Use inverted values from the blue channel ([code]1.0 - B[/code]).
+			[b]Alpha Inverted:[/b] Use inverted values from the alpha channel ([code]1.0 - A[/code]).
+			[b]Unused:[/b] Set the color channel's value to the default ([code]1.0[/code] for alpha, [code]0.0[/code] for red, green or blue).
+			[b]Zero:[/b] Set the color channel's value to [code]0.0[/code].
+			[b]One:[/b] Set the color channel's value to [code]1.0[/code].
+		</member>
```

Comment:
I think they're unavoidable though, since the main purpose of these property descriptions is to show up in an inspector tooltip:
![godot windows editor x86_64_miJUimUAnV](https://github.com/user-attachments/assets/975f8f26-0e28-4f56-b470-fbcae0d59529)

Changing the values to an enum and moving the full descriptions to the enum would mean that you have to open the docs and navigate away from the importer to see the descriptions of what each item does. Oh well.

Reply:
> Changing the values to an enum and moving the full descriptions to the enum would mean that you have to open the docs and navigate away from the importer to see the descriptions of what each item does. Oh well.

[We have something that handles this in the inspector](https://github.com/godotengine/godot/pull/76238), but not in the import options, project settings or editor settings yet.